### PR TITLE
BuildX failing for Docker CLI on Windows fixed

### PR DESCRIPTION
**Pull request by derHeinz**
Fix https://github.com/fabric8io/docker-maven-plugin/issues/1818

### CODE REVIEW

File: src/test/java/io/fabric8/maven/docker/service/BuildXServiceTest.java [hunk 1]
```diff
@@ -1,57 +1,43 @@
...
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.anyString;
-import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.mockConstruction;
-import static org.mockito.Mockito.mockStatic;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.*;
```
Comment (rohanKanojia) [inline src/test/java/io/fabric8/maven/docker/service/BuildXServiceTest.java:40]: Could you please revert these wildcard import changes?

Comment (derHeinz) [reply src/test/java/io/fabric8/maven/docker/service/BuildXServiceTest.java]: ok, just used vs-code to do so.

**Comment by derHeinz** [comment]
The failling checks are all some docker init problem...have nothing todo with the code-change.

File: src/main/java/io/fabric8/maven/docker/service/BuildXService.java [hunk 2]
```diff
@@ -5,36 +5,17 @@
 import io.fabric8.maven.docker.access.util.ExternalCommand;
 import io.fabric8.maven.docker.assembly.BuildDirs;
 import io.fabric8.maven.docker.assembly.DockerAssemblyManager;
-import io.fabric8.maven.docker.config.AttestationConfiguration;
-import io.fabric8.maven.docker.config.BuildImageConfiguration;
-import io.fabric8.maven.docker.config.BuildXConfiguration;
-import io.fabric8.maven.docker.config.ConfigHelper;
-import io.fabric8.maven.docker.config.ImageConfiguration;
-import io.fabric8.maven.docker.config.SecretConfiguration;
+import io.fabric8.maven.docker.config.*;
```
Comment (rohanKanojia) [inline src/main/java/io/fabric8/maven/docker/service/BuildXService.java:8]: Looks like we need to address https://github.com/fabric8io/docker-maven-plugin/pull/1819#discussion_r1723017084 here as well.

**Comment by rohanKanojia** [comment]
@derHeinz : Thanks for your PR. Changes look good to me 👍 

Could you please add a line to doc/changelog.md regarding this change?

File: src/main/java/io/fabric8/maven/docker/service/BuildXService.java (hunk 2 above)
Comment (derHeinz) [reply src/main/java/io/fabric8/maven/docker/service/BuildXService.java]: ok tried to revert that aswell

**Comment by derHeinz** [comment]
> @derHeinz : Thanks for your PR. Changes look good to me 👍
> 
> Could you please add a line to doc/changelog.md regarding this change?

done

**Pull request merged**